src/main/java/org/sagebionetworks/web/server/servlet/InitSessionServlet.java

```diff
@@ -47,18 +57,18 @@ public void service(ServletRequest arg0, ServletResponse arg1) throws ServletExc
 	@Override
 	public void doPost(final HttpServletRequest request, HttpServletResponse response) throws ServletException, IOException {
 
-		// return the Set-Cookie response with the session token
+		// return the Set-Cookie response with the access token
 		try {
 			String sessionJson = IOUtils.toString(request.getReader());
 			JSONObjectAdapter adapter = new JSONObjectAdapterImpl(sessionJson);
-			Session s = new Session(adapter);
-			String sessionToken = s.getSessionToken();
-			if (sessionToken == null || sessionToken.isEmpty()) {
-				sessionToken = WebConstants.EXPIRE_SESSION_TOKEN;
+			AccessTokenWrapper s = new AccessTokenWrapper(adapter);
```

Comment:
Use my AccessTokenWrapper instead of the backend Session object, but has the same key that I care about (sessionToken)